package.json

```diff
@@ -83,6 +92,8 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-test-loader": "^2.2.0",
+    "ember-cli-typescript": "^2.0.0-beta.2",
```

Comment:
This should be in `dependencies`, right?

Reply:
I don't think so? Isn't that the idea behind `prepublish`?

Reply:
The `prepublish` script generates `.d.ts` files in the appropriate places, but as of v2 the transpilation process (e.g. stripping out annotations, type-only imports, etc) now occurs via a Babel plugin. This makes us much better overall ecosystem citizens, but it also means ember-cli-typescript now operates much more like a normal preprocessor addon. If it's only in `devDependencies` for an addon, it'll only apply to that addon's dummy app.